components/google-cloud/google_cloud_pipeline_components/experimental/custom_job/custom_job_sample.ipynb

```diff
@@ -0,0 +1,348 @@
+    "```\n",
+    "\n",
+    "<dt><b>Args:</b></dt><dd><p>component_spec: The component spec (factory function) to run as aiplatform custom job.\n",
+    "display_name: Optional. The name of the custom job. If not provided the</p>\n",
+    "<div><p>component_spec.name will be used instead.</p>\n",
+    "</div>\n",
+    "<dl class=\"simple\">\n",
+    "<dt>replica_count: Optional. The number of replicas to be split between master</dt><dd><p>workerPoolSpec and worker workerPoolSpec. (master always has 1 replica).</p>\n",
+    "</dd>\n",
+    "<dt>machine_type: Optional. The type of the machine to run the custom job. The</dt><dd><p>default value is “n1-standard-4”.</p>\n",
+    "</dd>\n",
+    "<dt>accelerator_type: Optional. The type of accelerator(s) that may be attached</dt><dd><p>to the machine as per accelerator_count. Optional.</p>\n",
+    "</dd>\n",
+    "<dt>accelerator_count: Optional. The number of accelerators to attach to the</dt><dd><p>machine.</p>\n",
+    "</dd>\n",
+    "<dt>boot_disk_type: Optional. Type of the boot disk (default is “pd-ssd”). Valid</dt><dd><dl class=\"simple\">\n",
+    "<dt>values: “pd-ssd” (Persistent Disk Solid State Drive) or “pd-standard”</dt><dd><p>(Persistent Disk Hard Disk Drive).</p>\n",
+    "</dd>\n",
+    "</dl>\n",
+    "</dd>\n",
+    "</dl>\n",
+    "<p>boot_disk_size_gb: Optional. Size in GB of the boot disk (default is 100GB).\n",
+    "timeout: Optional. The maximum job running time. The default is 7 days. A</p>\n",
+    "<div><p>duration in seconds with up to nine fractional digits, terminated by ‘s’.\n",
+    "Example: “3.5s”</p>\n",
+    "</div>\n",
+    "<dl class=\"simple\">\n",
+    "<dt>restart_job_on_worker_restart: Optional. Restarts the entire CustomJob if a</dt><dd><p>worker gets restarted. This feature can be used by distributed training\n",
+    "jobs that are not resilient to workers leaving and joining a job.</p>\n",
+    "</dd>\n",
+    "<dt>service_account: Optional. Specifies the service account for workload run-as</dt><dd><p>account.</p>\n",
+    "</dd>\n",
+    "<dt>network: Optional. The full name of the Compute Engine network to which the</dt><dd><p>job should be peered. For example, projects/12345/global/networks/myVPC.</p>\n",
+    "</dd>\n",
+    "<dt>worker_pool_specs: Optional, worker_pool_specs for distributed training. this</dt><dd><p>will overwite all other cluster configurations. For details, please see:\n",
+    "<a class=\"reference external\" href=\"https://cloud.google.com/ai-platform-unified/docs/training/distributed-training\">https://cloud.google.com/ai-platform-unified/docs/training/distributed-training</a></p>\n",
+    "</dd>\n",
+    "</dl>\n",
+    "</dd>\n",
+    "<dt><b>Returns:</b></dt><dd><p>A Custom Job component correspoinding to the input component.</p>\n",
+    "</dd>\n",
+    "</dl>\n",
+    "</dd></dl>"
+   ],
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Install additional packages"
```

Comment:
i think this cell and "restart the kernel" is also common among all FPC

Reply:
Done.